src/cookies.njk

```diff
@@ -28,44 +29,113 @@ layout: layout-single-page.njk
+
       <form class="js-cookies-page-form">
 
-        <h2 class="govuk-heading-l">Essential cookies (strictly necessary)</h2>
-        <h2 class="govuk-heading-l">Analytics cookies (optional)</h2>
-
-          {{ govukRadios({
-            name: "analytics",
-            idPrefix: "radio-analytics",
-            fieldset: {
-              legend: {
-                text: "Do you want to accept analytics cookies?"
-              },
-              classes: "js-cookies-page-form-fieldset",
-              attributes: {
-                id: "analytics",
-                hidden: true
-              }
-            },
-            items: [
-              {
-                text: "Yes",
-                value: "yes"
-              },
-              {
-                text: "No",
-                value: "no"
-              }
+        <h2 class="govuk-heading-l govuk-!-margin-top-6">Essential cookies (strictly necessary)</h2>
+
+        <p class="govuk-body">We use an essential cookie to remember when you accept or reject cookies on our website.</p>
+
+        {{ govukTable({
+          firstCellIsHeader: true,
+          caption: "Essential cookies we use",
+          head: [
+            { text: "Name" },
+            { text: "Purpose"},
+            { text: "Expires"}
+          ],
+          rows: [
+            [
+              {text: "design-system-cookie-policy"},
+              {text: "Saves your cookie consent settings"},
+              {text: "1 year"}
+            ]
+          ]
+        }) }}
+
+        <h2 class="govuk-heading-l govuk-!-margin-top-6">Analytics cookies (optional)</h2>
+
+        <p class="govuk-body">We use Google Analytics software to understand how people use the GOV.UK Design System. We do this to help make sure the site is meeting the needs of its users and to help us make improvements.</p>
+
+        <p class="govuk-body">We do not collect or store your personal information (for example your name or address) so this information cannot be used to identify who you are.</p>
+
+        <p class="govuk-body">We do not allow Google to use or share our analytics data.</p>
+
+        <p class="govuk-body">Google Analytics stores information about:</p>
+
+        <ul class="govuk-list govuk-list--bullet">
+          <li>the pages you visit</li>
+          <li>how long you spend on each page</li>
+          <li>how you arrived at the site</li>
+          <li>what you click on while you visit the site</li>
+          <li>the device and browser you use</li>
+        </ul>
+
+        {{ govukTable({
+          firstCellIsHeader: true,
+          caption: "Analytics cookies we use",
+          head: [
+            { text: "Name" },
+            { text: "Purpose" },
+            { text: "Expires" }
+          ],
+          rows: [
+            [
+              { text: "_ga" },
+              { text: "Used to distinguish users." },
```

Comment:
Should we use [the same wording used on GOV.UK](https://www.gov.uk/help/cookie-details)?

```suggestion
              { text: "These help us count how many people visit GOV.UK by tracking if you’ve visited before" },
```

Reply:
@calvin-lau-sig7 What do you think about this? (sub "GOV.UK" with "GOV.UK Design System")

Reply:
Sure, that sounds sensible. If we'd like, can simplify 'tracking' a bit too?

```suggestion
              { text: "These help us count how many people visit the GOV.UK Design System by telling us if you’ve visited before." },
```

Reply:
GOV.UK use the same text for `_ga` and `_gid` - and the Google docs don't differentiate between the 2 either. Should we use the same "These help us count how many people visit the GOV.UK Design System by telling us if you’ve visited before." text for both?

Reply:
I chatted with @calvin-lau-sig7 about this, he's suggested "Helps us count how many people visit the GOV.UK Design System..." for both, I have updated the commit.